web/templates/user/user_create.html

```diff
@@ -15,28 +15,6 @@
 {% endblock %}
 
 {% block page_script %}
-    {{ resources.link_script_file('lazy-load-select.js' | require) }}
```

Comment:
The `lazy-load-select.js` is missing on this page.
I would propose to use the `generic_form` template for this form, where the script is already included.  Only the `confirmable-error.js` script must be added there.

Reply:
I think instead I will do the „clean“ version and include all form-input-features in a `form.js` bundle, which can then be included. We should do something similar with regards to `table`-related scripts (bootstrap table etc), and remove these from the `main` bundle to reduce unneccessary initial load.

Reply:
For this PR, I will just do a quickfix tho and re-add these references to not introduce unneccessary diff.